### PR TITLE
Add the ability to specify an alternative Object3D to the camera to apply the pose coming from a VRDevice

### PR DESCRIPTION
**Pull request by dmarcos**
…

### CODE REVIEW

File: src/renderers/webvr/WebVRManager.js [hunk 1]
```diff
@@ -85,6 +85,14 @@ function WebVRManager( renderer ) {
 
 	};
 
+	this.headObject = null;
+
+	this.setHeadObject = function ( object ) {
```
Comment (mrdoob) [inline src/renderers/webvr/WebVRManager.js]: `this.setPoseTarget`?

File: src/renderers/webvr/WebVRManager.js [hunk 2]
```diff
@@ -85,6 +85,14 @@ function WebVRManager( renderer ) {
 
 	};
 
+	this.headObject = null;
```
Comment (mrdoob) [inline src/renderers/webvr/WebVRManager.js]: Wouldn't make this one public...

**Comment by dmarcos** [comment]
Ready 👍

**Comment by mrdoob** [comment]
Should we also update the `camera.updateMatrixWorld();` line?

**Comment by dmarcos** [comment]
Probably. Should the camera be always updated as well?

**Comment by dmarcos** [comment]
I'm calling now to `poseObject.updateMatrixWorld` I believe we still need to call both. In A-Frame the camera is a child of the poseObject.

**Comment by dmarcos** [comment]
I know just call `poseObject.updateMatrixWorld()` that should also update the camera matrixWorld when it is a child of the poseObject

**Comment by mrdoob** [comment]
How do you guys handle `standing`?

https://github.com/mrdoob/three.js/blob/dev/src/renderers/webvr/WebVRManager.js#L119-L129

Should I get rid of that code? In the webvr examples I opted for adding a user `Group`.

https://github.com/mrdoob/three.js/commit/3cb51d868ee279be373815de592fc072deffc82c#diff-bfebe34154a0dfd9fc7b447fc9ed74e9

**Comment by dmarcos** [comment]
@mrdoob We have a 1.6 default height on 3DOF headsets to always simulate standing and we just apply whatever position comes from 6DOF headsets. We don't use that code so we're ok with getting rid of it.

**Comment by mrdoob** [comment]
Oks!

**Comment by mrdoob** [comment]
Thanks!

**Comment by dmarcos** [comment]
Thanks!

**Pull request merged**